tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py

```diff
@@ -0,0 +1,202 @@
+import pytest
+import logging
+import time
+from tests.manage.monitoring.conftest import *
```

Comment:
no need to import from conftest. It's being done automatically by pytest.
Also, in general, we should not import from one test module to another.